specification/azurestackhci/resource-manager/Microsoft.AzureStackHCI/stable/2022-12-01/updateSummaries.json

```diff
@@ -0,0 +1,463 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PackageVersionInfo"
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-identifiers": [
+            "version"
+          ]
+        },
+        "currentVersion": {
+          "description": "Current Solution Bundle version of the stamp.",
+          "type": "string"
+        },
+        "lastUpdated": {
+          "description": "Last time an update installation completed successfully.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "lastChecked": {
+          "description": "Last time the update service successfully checked for updates",
+          "type": "string",
+          "format": "date-time"
+        },
+        "healthState": {
+          "description": "Overall health state for update-specific health checks.",
+          "type": "object",
+          "$ref": "#/definitions/HealthState",
+          "x-ms-client-flatten": true
+        },
+        "healthCheckResult": {
+          "description": "An array of pre-check result objects.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrecheckResult"
+          },
+          "x-ms-client-flatten": true,
+          "x-ms-identifiers": [
+            "name"
+          ]
+        },
+        "healthCheckDate": {
+          "description": "Last time the package-specific checks were run.",
+          "type": "string",
+          "format": "date-time"
+        },
+        "state": {
+          "description": "Overall update state of the stamp.",
+          "type": "string",
+          "enum": [
+            "Unknown",
+            "AppliedSuccessfully",
+            "UpdateAvailable",
+            "UpdateInProgress",
+            "UpdateFailed",
+            "NeedsAttention",
+            "PreparationInProgress",
+            "PreparationFailed"
+          ],
+          "x-ms-enum": {
+            "name": "updateSummariesPropertiesState",
+            "modelAsString": true
+          }
+        }
+      }
+    },
+    "PackageVersionInfo": {
+      "description": "Current version of each updatable component.",
+      "type": "object",
+      "properties": {
+        "packageType": {
+          "description": "Package type",
+          "type": "string"
+        },
+        "version": {
+          "description": "Package version",
+          "type": "string"
+        },
+        "lastUpdated": {
+          "description": "Last time this component was updated.",
+          "type": "string",
+          "format": "date-time"
+        }
+      }
+    },
+    "HealthState": {
+      "type": "string",
+      "enum": [
+        "Unknown",
+        "Success",
+        "Failure",
+        "Warning",
+        "Error",
+        "InProgress"
+      ],
+      "x-ms-enum": {
+        "name": "HealthState",
+        "modelAsString": true
+      }
+    },
+    "PrecheckResult": {
```

Comment:
We have another definition of `PrecheckResult` in `updates.json` as well. They are not identical therefore our generator reports "duplicate schema" issue
If 
- these two `PrecheckResult` are the same schema, consider either unify them to let them be identical (not recommended), or introduce a common file to hold this kind of common models between different files (recommended), or define this only in one place, the other file reference the definition in that file (less recommended)
- these two `PrecheckResult` are totally different things but accidentally have the same name, please be sure to change the name of one of them.

Reply:
Updated  PrecheckResult at both places to use same schema

Reply:
@ArcturusZhang, thanks for the comment, and I helped to check that these two defination of "PrecheckResult" are identical, so I will approve this pr.